index.bs

```diff
@@ -478,9 +496,10 @@ Integration with Service Worker {#integration-sw}
 -------------------------------------------------
 
 In https://w3c.github.io/ServiceWorker/#dom-fetchevent-respondwith, replace 10.1 with the following
-item.
+items.
 
-1.  If |response| is not a `Response` object, or the result of performing a
+1.  If |response| is not a `Response` object, or _event_'s request's associated request's
+    [=request/mode=] is "`no-cors`" and the result of performing a
```

Comment:
I'd like to see what this algorithm looks like after #9 and #10 land. Can we get those in, and rebase this accordingly? It's hard to piece together otherwise.

Reply:
I rebased this change on top of #9. It is shown as a combined change on the review. https://github.com/mikewest/corpp/pull/11/commits/dcd0828442e39d9f50be1eef216d629041b2b8d6 shows the diff.

I believe #10 and #11 is almost independent each other because I stopped working on "process navigate a response" on this change.

Reply:
Now #9 is merged so hopefully this change is clear enough.